scripts/scraper/venv/lib/python3.12/site-packages/_pytest/_py/path.py

```
@@ -0,0 +1,1475 @@
+            meth = None
+            try:
+                meth = getattr(self, name)
+            except AttributeError:
+                if name[:3] == "not":
+                    invert = True
+                    try:
+                        meth = getattr(self, name[3:])
+                    except AttributeError:
+                        pass
+            if meth is None:
+                raise TypeError(f"no {name!r} checker available for {self.path!r}")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)